rai/client.go

```diff
@@ -1304,6 +1305,16 @@ func (c *Client) GetTransactionProblems(id string) ([]interface{}, error) {
 	return result, nil
 }
 
+func (c *Client) CancelTransaction(id string) (*TransactionAsyncCancelResponse, error) {
+	var result TransactionAsyncCancelResponse
```

Comment:
this parses even if there's no message field in the response, right?

Reply:
yep that's right